### PR TITLE
[Feature] Disable a search button in the conversation if EAR is enabled

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Disable a search button in the conversation if EAR is enabled

### Dependencies

https://wearezeta.atlassian.net/browse/ZIOS-13593

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift [hunk 1]
```diff
@@ -85,6 +85,7 @@ extension ConversationViewController {
         let button = UIBarButtonItem(icon: showingSearchResults ? .activeSearch : .search, target: self, action: action)
         button.accessibilityIdentifier = "collection"
         button.accessibilityLabel = "conversation.action.search".localized
+        button.isEnabled = !conversation.isMessagesEncrypted
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift:88]: I left a comment in the DataModel PR regarding the name.

Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+NavigationBar.swift]: Fixed

**Comment by johnxnguyen** [comment]
@typfel should we disabled the button or remove it completely?

**Comment by typfel** [comment]
> @typfel should we disabled the button or remove it completely?

I think disabling is fine.

**Comment by KaterinaWire** [comment]
> It would be good to add a test for this.

@johnxnguyen Done

**Pull request merged**